picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+    """
+    Class used to show picture on Table Row
+    """
+    def __init__(self, imagePath,parent=None):
+        super(ImgWidget1, self).__init__(parent)
+        pic = QPixmap(imagePath)
+        self.setPixmap(pic)
+
+class File_dialog(QDialog):
+    """
+    Main class used for the dialog
+    """
+    def __init__(self,wi,he,path,parent=None):
+        self.path = path
+        self.formats=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
+        self.formats_lower=[]
+        for i in self.formats:self.formats_lower.append(i.lower())
+        self.formats_show=["MP3","FLAC","OGG","M4A","WMA","WAV","MP4","AAC","flv"]
+        super(File_dialog,self).__init__(parent)
+        self.initUI(wi,he)
+
+    def initUI(self,wi,he):
+        self.moveBack = QPushButton(self)
+        self.moveBack.resize(35,30)
+        self.moveBack.setIcon(QIcon('../resources/img-src/moveBack24_24.png'))
+        self.moveBack.setIconSize(QSize(35,30))
+        self.moveBack.move(.17*wi,10)
+        self.moveBack.clicked.connect(self.moveBack_action)
+        self.moveNext = QPushButton(self)
+        self.moveNext.resize(35,30)
+        self.moveNext.setIcon(QIcon('../resources/img-src/moveNext24_24.png'))
+        self.moveNext.setIconSize(QSize(35,30))
+        self.moveNext.move(.17*wi+35,10)
+        self.moveNext.clicked.connect(self.moveNext_action)
+
+        self.selectBtn = QPushButton("Select It",self)
+        self.selectBtn.resize(100,30)
+        self.selectBtn.move(wi-0.09*wi,10)
+        self.selectBtn.clicked.connect(lambda:self.select_action(True))
+
+        self.selectBtn1 = QPushButton("Open Folder",self)
+        self.selectBtn1.resize(150,30)
+        self.selectBtn1.move(wi-.219*wi,10)
+        self.selectBtn1.clicked.connect(lambda:self.open_action(True))
+
+        self.lineedit = QLineEdit("Search here",self)
+        self.lineedit.resize(250,30)
+        self.lineedit.move(wi-.51*wi,10)
+        #self.lineedit.hide()
+        self.helpbtn = QPushButton(self)
+        self.helpbtn.resize(40,30)
+        self.helpbtn.setIcon(QIcon('../resources/img-src/search24_24.png'))
+        self.helpbtn.setIconSize(QSize(40,30))
+        self.helpbtn.move(wi-.32*wi,10)
+        #self.helpbtn.clicked.connect(self.lineedit.show)
+        self.helpbtn.clicked.connect(self.search_update)
+
+        self.lw=int(.15*wi)
+        self.l1w=wi-self.lw
+        self.listWidget = QListWidget(self)
+        self.listWidget.resize(self.lw,he-.195*he)
+        self.listWidget.move(0,0)
+
+        self.table = QTableWidget(self)
+        self.table.setColumnCount(3)
+        self.table.setHorizontalHeaderLabels(("Name;Size;Modified").split(";"))
+        self.table.verticalHeader().setVisible(False)
+        self.table.setEditTriggers(QAbstractItemView.NoEditTriggers)
+        self.table.setSelectionBehavior(QAbstractItemView.SelectRows)
+        cw=int(.7*self.l1w)
+        cw1=int(.143*self.l1w)
+        cw2=int(.143*self.l1w)
```

Comment:
Avoid such arbitrary variable names. Use something verbose

Reply:
Variable names should be self-explanatory, those are clearly not.